api/client/ffs.go

```diff
@@ -22,48 +23,55 @@ type JobEvent struct {
 	Err error
 }
 
-// PushConfigOption mutates a push configuration.
-type PushConfigOption func(o *PushConfig)
+// NewAddressOption is a function that changes a NewAddressConfig
+type NewAddressOption func(r *rpc.NewAddrRequest)
```

Comment:
Found these options functions can operate on a rpc request nicely, no need for a separate config struct.